feat(merge): allow to prioritize queue (#1106)
Mergifyio/mergify-engine: doc/source/actions.rst

       Note that the ``rebase`` method has many drawbacks, see :ref:`strict
       rebase`.

   * - ``priority``

>      - positive integer or ``low`` or ``medium`` or ``high``

Reviewer: ```suggestion
     - 0 <= integer <= 10000 or ``low`` or ``medium`` or ``high``
```

>      - positive integer or one of ``low``, ``default``, ``high``

Reviewer: ```suggestion
     - positive integer or``low`` or ``medium`` or ``high``
```

`default` does not sound like it's between `high` and `low` and I'm sure that what you meant!

>      - `default`

Reviewer: ```suggestion
     - ``medium``
```

>      - `default`

Reviewer: ```suggestion
     - ``medium``
```

     - This set the priority of the pull request in the queue when ``smart``

>        :ref:`strict merge` is enabled. Higher number get merged first.

Reviewer: ```suggestion
       :ref:`strict merge` is enabled. The pull request with the highest priority is merged first.
```

>      - This set the priority of the pull request in the queue when ``smart``
>        :ref:`strict merge` is enabled. Higher number get merged first.
>        ``low``, ``default``, ``high`` are aliases for ``0``, ``10``, ``100``

Reviewer: ```suggestion
     - Sets the priority of the pull request in the queue when ``smart``
       :ref:`strict merge` is enabled. Higher numbers get merged first.
       ``low``, ``medium``, ``high`` are aliases for ``0``, ``10``, ``100``
```

I'm a bit disappointed you did not use emoji for this. Can we add some? 

- 🐌
- 🐢
- 🐇
- 🚲
- 🏎
- 🚀

🤣

>        ``low``, ``default``, ``high`` are aliases for ``1000``, ``2000``, ``3000``

Reviewer: ```suggestion
       ``low``, ``default``, ``high`` are aliases for ``1000``, ``2000``, ``3000``.
```
